mealy/error_analyzer.py

```diff
@@ -329,8 +345,7 @@ def _get_leaf_selector(self, leaf_selector):
                   * int: Only keep the row corresponding to this leaf id
                   * array-like: Only keep the rows corresponding to these leaf ids
                   * str:
-                    - "all": Keep the whole array
-                    - "all_errors": Keep the rows with indices corresponding to the leaf ids classifying the primary model prediction as wrong
+                    - "all": Keep the whole array of leaf ids
```

Comment:
Useless to keep a string if it can only take one value
```suggestion
                  * None: Keep the whole array of leaf ids
```